src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const lines = sourceString.split(';').map(line => line.trim());
+
+  lines.forEach(line => {
+    if (line) {
+      const [property, value] = line.split(':').map(str => str.trim());
```

Comment:
You don't need to split your code and put the blank line between. Just remember:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable.

Also, let's use reduce method to generate stylesObject, it's a better fit in this case.